postgresql/src/storages/postgres/detail/connection_impl.cpp

```diff
@@ -489,8 +525,7 @@ ResultSet ConnectionImpl::PortalExecute(
   UASSERT_MSG(prepared_info,
               "Portal execute uses statement id that is absent in prepared "
               "statements");
-
-  tracing::Span span{scopes::kQuery};
+  tracing::Span span{scopes::kExec + ":" + portal_name};
```

Comment:
I'd say we don't want to move business logic names to system span names. It is OK to leave them in span attributes, but don't bring them to span names. SQL command names like INSERT, DELETE, etc. is OK to move to span names, though.

Reply:
Ok, I'll handle them the same way as ordinal queries